install/bootstrap_livecd_usb.md

```diff
@@ -262,20 +263,20 @@ Some files are needed for generating the configuration payload. See these topics
    - `hmn_connections.json`
    - `ncn_metadata.csv`
    - `switch_metadata.csv`
-   - `system_config.yaml` (see below)
+   - `system_config.yaml` (only available after first-install generation of system files)
 
    > The optional `application_node_config.yaml` file may be provided for further defining of settings relating to how application nodes will appear in HSM for roles and subroles. See [Create Application Node YAML](create_application_node_config_yaml.md)
 
    > The optional `cabinets.yaml` file allows cabinet naming and numbering as well as some VLAN overrides. See [Create Cabinets YAML](create_cabinets_yaml.md).
 
    > The `system_config.yaml` is required for a reinstall, because it was created during a previous install. For a first time install, the information in it can be provided as command line arguments to `csi config init`.
 
-   After gathering the files into this working directory, generate your configurations:
+   After gathering the files into this working directory, generate your configurations
 
-1. If doing a reinstall and the `system_config.yaml` parameter file is available, then generate the system configuration reusing this parameter file (see [avoiding parameters](../background/cray_site_init_files.md#save-file--avoiding-parameters)).
-
-   If not doing a reinstall of Shasta software, then the `system_config.yaml` file will not be available, so skip the rest of this step.
+1. **For second-time installs (re-installs) where the `system_config.yaml`  parameter file is available**,generate the updated system configuration (see [avoiding parameters](../background/cray_site_init_files.md#save-file--avoiding-parameters)).
```

Comment:
maybe just "For subsequent installs (re-installs)" ?

Reply:
"subsequent fresh-installs (re-installs)"?